org.recoll.recoll.yml

```diff
@@ -0,0 +1,125 @@
+  - --device=dri
+  - --filesystem=home
+  - --env=ASPELL_CONF=dict-dir /app/share/dicts; local-data-dir /app/share/datadicts
+
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+
+modules:
+  - name: xapian-core
+    config-opts:
+      - --disable-sse
+      - --disable-backend-chert
+      - --disable-backend-remote
+      - --disable-documentation
+    cleanup:
+      - /bin
+      - /share/aclocal
+      - /share/doc
+      - /share/man
+    sources:
+      - type: archive
+        url: https://oligarchy.co.uk/xapian/1.4.26/xapian-core-1.4.26.tar.xz
+        sha256: 9e6a7903806966d16ce220b49377c9c8fad667c8f0ffcb23a3442946269363a7
+
+  - name: chmlib
+    cleanup:
+      - /include
+      - '*.a'
+    sources:
+      - type: archive
+        url: http://www.jedrea.com/chmlib/chmlib-0.40.tar.bz2
+        sha256: 3449d64b0cf71578b2c7e3ddc048d4af3661f44a83941ea074a7813f3a59ffa3
+      - type: script
+        commands:
+          - autoreconf --install
+
+  - name: dictEn-aspell
+    buildsystem: simple
+    build-commands:
+      - ./configure
+      - make
+      - make dictdir=/app/share/dicts datadir=/app/share/datadicts install
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/aspell/dict/en/aspell6-en-2020.12.07-0.tar.bz2
+        sha256: 4c8f734a28a088b88bb6481fcf972d0b2c3dc8da944f7673283ce487eac49fb3
+
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --prefix=/app
+      - ./b2 install
+    cleanup:
+      - '*'
+    sources:
+      - type: archive
+        url: https://archives.boost.io/release/1.86.0/source/boost_1_86_0.tar.bz2
+        sha256: 1bed88e40401b2cb7a1f76d4bab499e352fa4d0c5f31c0dbae64e24d34d7513b
+
+  - name: poppler
+    buildsystem: cmake-ninja
+    cleanup:
+      - /share
+    config-opts:
+      - -DENABLE_CPP=OFF
+      - -DENABLE_GLIB=OFF
+      - -DENABLE_QT5=OFF
+      - -DENABLE_QT6=OFF
+    sources:
+      - type: archive
+        url: https://poppler.freedesktop.org/poppler-24.11.0.tar.xz
+        sha256: 7723d880565211740c13649d24a300257b86ddd7fa2d208187ff7e5cc8dfbd58
+
+  - name: perl
+    cleanup:
+      - /share/man
+      - '*.h'
+    sources:
+      - type: archive
+        url: https://www.cpan.org/src/5.0/perl-5.40.0.tar.gz
+        sha256: c740348f357396327a9795d3e8323bafd0fe8a5c7835fc1cbaba0cc8dfe7161f
+      - type: script
+        dest-filename: configure
+        commands:
+          - ./Configure -des -Dprefix=/app
+    post-install:
+      - find /app/lib/perl5 -type f -exec chmod u+w {} \;
+
+  - name: exiftool
+    buildsystem: simple
+    build-commands:
+      - perl Makefile.PL
+      - make install
+    cleanup:
+      - /share/man
+    sources:
+      - type: archive
+        url: https://exiftool.org/Image-ExifTool-13.02.tar.gz
+        sha256: b98ce4f610b7ca8b4989b94f12e1f4f40d4aa2b364d9c2915e97f41dc0f5d008
+
+  - name: recoll
+    buildsystem: meson
+    build-options:
+      env:
+        QMAKEPATH: /app/lib
```

Comment:
why do you need QMAKEPATH with meson?

Reply:
It is unable to locate the `webkitwidgets` module without this environment variable:
```
[124/127] Generating qtgui/recoll with a custom command
FAILED: qtgui/recoll 
/run/build/recoll/qtgui/qmkmk.sh qtgui/recoll.pro PREFIX=/app
Info: creating stash file /run/build/recoll/_flatpak_build/qtgui/.qmake.stash
Project MESSAGE: Prefix is /app
Project ERROR: Unknown module(s) in QT: webkitwidgets
```